### ISSUE TITLE
`visitor-counter` integration

### ISSUE THREAD

**Issue by ghost** [erikthalen/visitor-counter#8]
Hi @erikthalen ,

Now that you made a UI, I think this is intended to be integrated as a nice plugin on famous web libraries (Express and brothers)

For this, they have "sometimes" minor differences calling native node `http` objects. I for instance modified this to be able to integrate as a plugin in Fastify

https://github.com/bacloud22/visitor-counter/blob/44ded9b749044119f35ea05cce6919e85bf5ac62/lib/index.mjs#L24

So in [Fastify](https://www.fastify.io/docs/latest/Reference/Reply/) for instance they provide `req.raw`, `reply.raw` to have access to `http` native API. (.raw - The [http.ServerResponse](https://nodejs.org/dist/latest-v14.x/docs/api/http.html#http_class_http_serverresponse) from Node core).

Anyway, this is just a hint for operations on Request object when setting cookie or dealing with headers, etc, etc. If you want to support more libraries (koa, Fastify..) minor safe guards and you are done.

**Comment by erikthalen** [comment]
Yeah, would be cool to be able to specify framework somewhere. Something like:
```js
const stats = await visitorCounter({
  env: 'fastify'
})
```
And then have `visitor-counter` call different functions depending on which is needed. I've only tried it with Express so far, which works out-of-the-box.

Haven't used f.ex. fastify. I'm happy to merge your change, if there's a way to have it work with that to!

**Comment by ghost** [comment]
I will try it again (was deactivated) while I'm developing with Fastify. So besides [setting the header `set-cookie`](https://github.com/bacloud22/visitor-counter/blob/44ded9b749044119f35ea05cce6919e85bf5ac62/lib/index.mjs#L24) that was buggy, I will propose a PR if anything else raises.
Good luck!

**Comment by bacloud22** [comment]
Meanwhile, if you talk about `visitor-counter` on dev.to, I'm sure a lot of developers would appreciate this.
